model_analyzer/analyzer.py

```diff
@@ -423,5 +423,14 @@ def _warn_if_other_models_loaded_on_remote_server(self, client):
     def _populate_search_parameters(self):
         for model in self._config.profile_models:
             self._search_parameters[model.model_name()] = SearchParameters(
-                self._config, model.parameters(), model.model_config_parameters()
+                self._config,
+                model.parameters(),
+                model.model_config_parameters(),
+                is_bls_model=bool(self._config.bls_composing_models),
+            )
+        for model in self._config.bls_composing_models:
+            self._search_parameters[model.model_name()] = SearchParameters(
```

Comment:
Is there ever an instance where the composing model can have the same name as a higher level model?
Seems like you would overwrite an entry in that case?

Reply:
Good catch! This is why I have separate models and composing_models in the code. I've updated this logic to do the same.